src/i18n/index.ts

```diff
@@ -41,6 +44,7 @@ export async function initI18n(
     cachedTranslations = await getOtaTranslations()
   }
   const resources = getAvailableResources(cachedTranslations)
+  const { links } = getDynamicConfigParams(DynamicConfigs[StatsigDynamicConfigs.APP_CONFIG])
```

Comment:
I noticed this is loaded before statsig is initialized, so I think this just falls back to the default in the wallet